packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownBestPractices.md

```diff
@@ -0,0 +1,13 @@
+### Layout
+
+- Use a drop-down list when there are multiple choices that can be collapsed under one title, if the list of items is too long, or when space is constrained.
+- Use a drop-down list when the selected option is more important than the alternatives (in contrast to radio buttons where all the choices are visible, putting equal emphasis on all options).
+
+### Content
+
+- Use sentence-style capitalization—only capitalize the first word. For more info, see [Capitalization](https://docs.microsoft.com/style-guide/capitalization) in the Microsoft Writing Style Guide.
```

Comment:
```suggestion
- Use sentence-style capitalization — only capitalize the first word. For more info, see [Capitalization](https://docs.microsoft.com/style-guide/capitalization) in the Microsoft Writing Style Guide.
```

Reply:
This is an em dash, so no spacing on either side (per Microsoft Writing Style Guide).